### ISSUE TITLE
inability to fully extend finger from base (phalangeal / metacarpal joint)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10700]
**Preferred term label:**
inability to fully extend finger from base (phalangeal / metacarpal joint)

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mallet finger
Extensor tendon laceration, hand/wrist

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am not an expert, but this definition does not seem to match the description of Mallet finger, please check!

https://pubmed.ncbi.nlm.nih.gov/16148359/

**Comment by MickeySegal** [comment]
We now have this finding for 4 diseases, including also Posterior interosseous nerve syndrome and Hand high pressure injection injury.  For mallet finger we of course list "Flexion at finger distal interphalangeal (DIP) joint" but we also list "inability to fully extend finger from base (phalangeal / metacarpal joint)" because the cause is often a distal extensor tendon rupture and leads to both findings.

BTW, for "Flexion at finger distal interphalangeal (DIP) joint" we used https://hpo.jax.org/browse/term/HP:0012785 for which there are more granular findings for each finger, but it would be good to have a new finding based on the affected joint.

**Comment by pnrobinson** [comment]
@MickeySegal 
Not sure if additional terms are required, but please open new issue if so. 

Adding new term: 
- ID: HP:6001196
- label: Inability to fully extend finger at metacarpophalangeal joint
- parents: [Decreased finger mobility (HP:0006135)](https://hpo.jax.org/app/browse/term/HP:0006135)
- pmid: PMID:39345665
- def: An inability to fully extend the metacarpophalangeal (MCP) joint of a finger (extension of an MCP joint straightens the finger).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10700
- synonyms: Inability to fully extend finger at MCP joint(NONE)|Inability to fully extend finger from base(NONE)
